plugins/announcements/src/components/AnnouncementsPage/AnnouncementsPage.tsx

```diff
@@ -294,6 +294,7 @@ type AnnouncementsPageProps = {
   subtitle?: ReactNode;
   maxPerPage?: number;
   category?: string;
+  createNoun?: string;
   cardOptions?: AnnouncementCardProps;
```

Comment:
Let's do something similar to what we did for `cardOptions`. I'm thinking

```ts
type AnnouncementCreateButtonProps = {
  name?: string
}

type AnnouncementsPageProps = {
  ...,
  buttonOptions?: AnnouncementCreateButtonProps
}
```

What are your thoughts?